src/media/css/header.styl

```diff
@@ -346,7 +346,11 @@ body > .banners mkt-banner {
 
 @media $base-tablet {
     .subheader {
-        h1,
+        h1 {
```

Comment:
This change results in nothing. The comma operator was already ensuring the two targets share the same properties. Maybe you had a different change that was missed in the commit?